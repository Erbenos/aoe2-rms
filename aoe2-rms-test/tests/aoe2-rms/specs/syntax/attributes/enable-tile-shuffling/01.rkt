#lang aoe2-rms
; Create 4 individual gold mines randomly positioned in a circle around players, with no bias towards any position.

<OBJECTS-GENERATION>
(create-object 'GOLD
               (set-place-for-every-player)
               (set-gaia-object-only)
               (number-of-objects 4)
               (min-distance-to-players 12)
               (set-circular-placement)
               (find-closest)
               (enable-tile-shuffling))
