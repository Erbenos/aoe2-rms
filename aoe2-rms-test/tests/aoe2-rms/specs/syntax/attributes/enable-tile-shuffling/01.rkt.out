<OBJECTS_GENERATION>
create_object GOLD {
  set_place_for_every_player
  set_gaia_object_only
  number_of_objects 4
  min_distance_to_players 12
  set_circular_placement
  find_closest
  enable_tile_shuffling
}
